gamersnet_backend/persistence/users.js

```diff
@@ -11,6 +12,13 @@ async function connect() {
     users = db.collection('users');
 }
 
+async function getUserByID(id) {
+    await connect();
+
+    let result = users.findOne({_id: new ObjectID(id)});
```

Comment:
may wanna remove the 'new', i think it creates new Object id.. i might be wrong though. Recheck it.

Reply:
It does not appear to increment the ObjectID counter. Tested using the following script:
```javascript
'use strict';

let ObjectID = require('mongodb').ObjectID;

let obj1 = new ObjectID();
console.log('obj1 ID', obj1.id);

console.log('======');

let obj2 = new ObjectID(obj1.id);
console.log('obj1 ID (after new)', obj1.id);
console.log('obj2 ID', obj2.id);

console.log('======');

let obj3 = new ObjectID();
console.log('obj3 ID', obj3.id);
```
```
obj1 ID <Buffer 60 59 52 24 7b ed b6 00 6d 2a d8 15>
======
obj1 ID (after new) <Buffer 60 59 52 24 7b ed b6 00 6d 2a d8 15>
obj2 ID <Buffer 60 59 52 24 7b ed b6 00 6d 2a d8 15>
======
obj3 ID <Buffer 60 59 52 24 7b ed b6 00 6d 2a d8 16>
```